docker-compose.yml

```diff
@@ -28,18 +28,6 @@ services:
       - SECRET_KEY=dev
       - AUTHORIZATION_TOKEN=auth-token
       - GET_AUTHORIZATION_TOKEN=get-auth-token
-      - UWSGI_HTTP=0.0.0.0:8001
-      - UWSGI_MASTER=1
-      - UWSGI_STATIC_INDEX=index.html
```

Comment:
deze mist in de nieuwe config. Maar was misschien overbodig?

Reply:
Alles tests succeeded, dus deze lijkt idd overbodig.